postgrenosql/src/main/java/com/yahoo/ycsb/db/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017 YCSB Contributors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+/**
+ * The YCSB binding for PostgreNoSQL client.
+ */
+package com.yahoo.ycsb.db;
```

Comment:
please place these classes into a binding-specific package, i.e. `com.yahoo.ycsb.postgresnosql`

Reply:
@busbey: It is not really consistent among the different projects. Some add the clients into a new package (Couchbase2) as you suggested, but others (Cassandra, MongoDB) use the com.yahoo.ycsb.db package. In my first implementation I used an own package but changed it due the feedback I received. So what is the right way?